tests/test_poincare_math.py

```diff
@@ -370,3 +370,8 @@ def test_distance2plane(a, c):
     dist = poincare.math.dist2plane(z, a, vr, c=c)
 
     np.testing.assert_allclose(dist, dist1, atol=1e-5, rtol=1e-5)
+
+def test_klein_transition(a, c):
```

Comment:
Do you know any test for midpoint?

Reply:
I wasn't able to think of one. I can check with zero curvature.